packages/shared/src/components/Feed.tsx

```diff
@@ -305,7 +305,7 @@ export default function Feed<T>({
     );
   };
 
-  const onAdClick = (ad: Ad, index: number, row: number, column: number) => {
+  const onAdClick = (ad: Ad, _: number, row: number, column: number) => {
```

Comment:
If we don't need the number, can we not send it as a parameter?

Reply:
Yeah sure, didn't properly notice